README.md

```diff
@@ -15,7 +15,7 @@ test commita
 
 Test a commit ddThe standard chunk of Lorem Ipsum used since the 1500s is reproduced below for those interested. Sections 1.10.32 and 1.10.33 from "de Finibus Bonorum et Malorum" by Cicero are also reproduced in their exact original form, accompanied by English versions from the 1914 translation by H. Rackham.
```

Comment:
@apurva1112 
2nd paragraph. *Italic*, **bold**, and `monospace`. Itemized lists
look like:
LETS ROCK
  * this one
  * that one
Use 3 dashes for an em-dash. Use 2 dashes for ranges (ex., "it's all
in chapters 12--14"). Three dots ... will be converted to an ellipsis.
Unicode is supported. ☺
mellow

DEMO DEMO